react/react1/week1/Todo/src/components/TodoListItem/TodoListItem.jsx

```diff
@@ -0,0 +1,12 @@
+import './TodoListItem.css'
+
```

Comment:
Good use of component here. Good to see you do styling also

Reply:
Hi, Lucas. Thack you for review )